### DISCUSSION TITLE
Docs Improvements: Settings and Running the ETL

### DISCUSSION THREAD

**Discussion by aesharpe** [catalyst-cooperative/pudl#1643] [Ideas]
I think it would improve the flow of the docs to embed the **[Development/Settings](https://catalystcoop-pudl.readthedocs.io/en/latest/dev/settings_files.html)** docs page in the **[Development/Running The ETL](https://catalystcoop-pudl.readthedocs.io/en/latest/dev/run_the_etl.html)** page. It seems a little confusing to read about the settings before you understand the context of the scripts you're creating settings for. My proposal would be to remove the **Settings** page and embed that information under a sub-header in the **Running The ETL** page. What do people think?

**Comment by zaneselvans** [comment]
Sure that seems reasonable. Neither of them is ridiculously long right now.

**Comment by aesharpe** [comment]
:sparkles: Done! Checkout my PR #1642
